src/libstd/io/buffered.rs

```diff
@@ -293,7 +293,7 @@ impl<R: Seek> Seek for BufReader<R> {
     /// where `n` minus the internal buffer length overflows an `i64`, two
     /// seeks will be performed instead of one. If the second seek returns
     /// `Err`, the underlying reader will be left at the same position it would
-    /// have if you seeked to `SeekFrom::Current(0)`.
+    /// have if you had gotten to `SeekFrom::Current(0)`.
```

Comment:
cc @varkor ↑

Reply:
Sorry, didn't spot the change! I think something like: ```/// have if you called `seek` with `SeekFrom::Current(0)`.``` would be more clear, as it describes the intention (seeking *to* `SeekFrom::Current(0)`), rather than the implementation's behaviour.